axelrod/strategies/axelrod_second.py

```diff
@@ -1550,7 +1550,7 @@ def strategy(self, opponent: Player) -> Action:
 class Black(Player):
     """
     Strategy submitted to Axelrod's second tournament by Paul E Black (K83R)
-    and came in fourteenth in that tournament.
+    and came in fifteenth in that tournament.
```

Comment:
As an FYI, this is in essence stepping on #1161. No problem at all (don't change it) but just a headsup.